src/db/mormot.db.raw.oracle.pas

```diff
@@ -897,6 +897,8 @@   TSqlDBOracleLib = class(TSynLibrary)
   public
     // the client verion numbers
     major_version, minor_version, update_num, patch_num, port_update_num: sword;
+    /// Maximum supported varchar string length

```

Comment:
Do not use "varchar string length", but just "varchar bytes".
My guess is that it should be part of the oracle driver properties, not at the OracleLib level.

Reply:
Ok.